tardis/montecarlo/tests/test_montecarlo.py

```diff
@@ -238,28 +238,6 @@ def test_binary_search(x, x_insert, imin, imax, expected_params):
     assert obtained_result == expected_params["result"]
 
 
-@pytest.mark.parametrize(
-    ["mu", "r", "inv_t_exp", "expected"],
-    [
-        (0.3, 7.5e14, 1 / 5.2e7, 0.9998556693818854),
-        (-0.3, 0, 1 / 2.6e7, 1.0),
-        (0, 1, 1 / 2.6e7, 1.0),
-    ],
-)
-def test_get_doppler_factor(mu, r, inv_t_exp, expected):
-    # Set the params from test cases here
-    # TODO: add relativity tests
-    time_explosion = 1 / inv_t_exp
-
-    # Perform any other setups just before this, they can be additional calls
-    # to other methods or introduction of some temporary variables
-
-    obtained = get_doppler_factor(r, mu, time_explosion)
-
-    # Perform required assertions
-    assert_almost_equal(obtained, expected)
-
-
```

Comment:
There are more doppler factor tests below. I would also move them to the new file.

Reply:
I agree. It looks like it doesn't exist anymore. I also just noticed that all tests in test_montecarlo.py are skipped because of `pytestmark = pytest.mark.skip(reason="Port from C to numba")`. This explains why the tests did not fail.

Reply:
Can you make an issue that we should check if there are any tests in test_montecarlo.py that we actually need but are currently skipped?